### ISSUE TITLE
Can't search for query containing a space

### ISSUE THREAD

**Issue by zach-is-my-name** [trufflesuite/trufflesuite.com#917]
![Screenshot from 2021-02-20 00-37-05](https://user-images.githubusercontent.com/16063092/108585052-1bed2a80-7314-11eb-9380-c255408371bf.png)

**Comment by gnidan** [comment]
No, no you cannot! Oops. We'll get this into our backlog to revisit our search capabilities. Thank you for reporting this.
